examples/main.tf

```diff
@@ -41,21 +41,21 @@ resource "propel_data_source" "http_data_source" {
   }
 }
 
-resource "propel_data_source" "webhook_data_source" {
-  unique_name = "My Webhook Data Source"
+resource "propel_data_source" "my_webhook_data_source" {
+  unique_name = "MyWebhookDataPool"
```

Comment:
We don't have to change the unique names. They can have spaces

Reply:
I simply relocated the example that was in the generated file but not in the source. As a result, when we ran the command to regenerate the docs, that example was removed.

Changed the unique name as suggested